docs/diginstroom/sip/2.0/profiles/material-artwork.md

```diff
@@ -0,0 +1,185 @@
+         └──preservation
+            └── premis.xml
+   └──representation_4       # quality assessment reference
+      │── mets.xml
+      └──data
+      |  |── PID_REF_OBJ.OBJ               # polygon file    
+      |  |── PID_REF_BMP.BMP               # texture image      
+      |  |── PID_REF_IJK_BMP.BMP           # reference texture image     
+      │  └── PID_REF_MTL.MTL               # texture mapping file
+      │
+      └──metadata
+         |── descriptive     (optional)
+         |   └── dc+schema.xml    
+         └──preservation
+            └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- There MUST be exactly one IE at the root. There MAY be other sub-IE's that are part of the root IE or other sub-IE's (e.g. to describe panels of a triptych individually).
+- There MUST be at least one representation, but there MAY be multiple: either containing different captures or resolutions of the same IE or representing a different IE.
+- Each representation MUST contain at least one file. 
+- Preservation metadata MUST be limited to the PREMIS metadata schema.
+- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
+- There MUST be preservation metadata at the representation level in the `preservation/premis.xml` file.
+- Only the MD5 hashing algorithm is allowed to compute the fixity, thus:
+  - The value of element `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm` MUST be set to `MD5`.
+  - The value of attribute `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm/@valueURI` MUST be set to `"http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5"`.
+
+### Package METS
+
+- The `csip:CONTENTINFORMATIONTYPE` attribute MUST be set to `OTHER` and the `csip:OTHERCONTENTINFORMATIONTYPE` attribute MUST be set to `https://data.hetarchief.be/id/sip/2.0/material-artwork`.
+- The `TYPE` attribute in the `mets.xml` file MUST be set to
+  - `Photographs - Digital` (for 2D objects) or
+  - `Scanned 3D Objects (output from photogrammetry scanning)` (for 3D objects).
+- The `mets/dmdSec/mdRef/@MDTYPE` attribute MUST be set to `OTHER` and `mets/dmdSec/mdRef/@OTHERMDTYPE` attribute must be set to `DC+SCHEMA`.
+  
+
+### Descriptive Metadata
+
+- A descriptive metadata file `descriptive/dc+schema.xml` describing the IE MUST be present at the package level.
+- A descriptive metadata file `descriptive/dc+schema.xml` describing the representation MAY be present at the representation level (eg. to indicate diverting licenses). 
+- Descriptive metadata in `dc+schema.xml` MUST be limited to the [DCTERMS](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) and [SCHEMA](http://schema.org) elements outlined in the [basic profile]({{ site.baseurl }}{% link docs/diginstroom/sip/2.0/profiles/basic.md %}#dc-requirements).
```

Comment:
What is the big difference between the basic and this profile then, is this one is limited to the same descriptive metadata?